mailboxes/folders-lib.pl

```diff
@@ -2494,14 +2494,15 @@ sub safe_uidl
 # Attempts to convert some HTML to text form
 sub html_to_text
 {
-local ($h2, $lynx);
+my ($html) = @_;
+my ($h2, $lynx);
 if (($h2 = &has_command("html2text")) || ($lynx = &has_command("lynx"))) {
 	# Can use a commonly available external program
 	local $temp = &transname().".html";
 	open(TEMP, ">", $temp);
-	print TEMP $_[0];
+	print TEMP $html;
 	close(TEMP);
-	open(OUT, ($lynx ? "$lynx -dump $temp" : "$h2 $temp")." 2>/dev/null |");
+	open(OUT, ($lynx ? "$lynx --display_charset=utf-8 -dump $temp" : "$h2 $temp")." 2>/dev/null |");
```

Comment:
Unrelated to this PR, but we should really remove this ancient use of `lynx` to do HTML conversion!

Reply:
> `lynx` actually is really good at doing the job! 

.. better than `html2text` and also better than `HTML::TreeBuilder` and `HTML::FormatText`..